modules/features/player/src/main/res/layout-sw600dp/adapter_player_header.xml

```diff
@@ -0,0 +1,378 @@
+<?xml version="1.0" encoding="utf-8"?><!-- Large screens use default layout instead of the landscape one to match iOS design.
+Player controls and seekbar are centered horizontally on this layout, which makes them easier to transition to the Transcript screen.  -->
+<au.com.shiftyjelly.pocketcasts.views.component.LockableNestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This is a duplicate of the portrait mode layout. I added a comment above about why we need it. I tried using [layout aliases](https://stackoverflow.com/a/10019406/193545) to not duplicate it but ViewBinding doesn't work with it.